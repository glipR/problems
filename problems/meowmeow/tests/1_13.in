65
tril
hissssssssssssssssssssssss
moiprrnaltrillllllllllllllllllllllllllllh
his
his
buuuuuuuuurrrrrrrdrrrrrrrrrblllllllllllle
aigezs
xwpgknlxyd
qoqo
hss
burbe
brblle
hissssssssssssj
buuuuuuuuuuuurrrrbllllllllllllle
buuuuuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrbllllllllllllllllllllle
u
laelwwitwyhibsss
xwidzhnjijhis
ktgekkzpcwhtlaapagdb
sllrshissssssssssssssssssssssssss
buuuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrbllllllllllllllllllllllle
burrrrrrrrrblllllllllllllllllllllled
hisssssssssssssssssssssssssssssssss
uggykhis
triiiiiiiiiiiiiiiiiiiiilllllllllllllllll
sklkrqwkbbubqegzmkb
pjtvmychzqvho
buuuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrrrbllllllllllllllle
mybxpyleno
buusrrrrrrrrrrrrrrrrrrbllllllllllllllllllle
jqvngnxjexmjppepkel
trll
hissssssssssssssssssssssssss
zapkharaqhrdlurxhjco
tril
qgllfhfxga
tril
buuuuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrbllllllllllllllllle
triiiiiiiiiiiiiiiiiiiiiiillllllllllllllllllllllxll
ipfinfvrwhss
trll
vv
hisssssse
uxlxbinwddtridbn
triiiiiiiiilll
bdkbntondkiyyz
lkthissssssssssssssssssssssssssssssssssssssssssssk
hissssssss
triiiiiiiiiiiiiiiiiiiiiilllllllllllllllllllllllllll
hisssssssssssssssssssskssssssssssssss
triiiiiiiiiiiiiiiiiiiiiiiiiiillllllllllllllllllllllllllll
brblle
bhcxiwhvh
nmludbzobuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrbllllllllllllllllllllle
trll
cviifgrjczxil
zsei
hisssssssssssssssssssssssssssss
hisssssssssssssz
buuuuuuuuuuuuuuuuuuurrrrrrrrrrbllllllllllllle
buuuuuuuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrbllllllllllllllllles
his
hissssssm
zwkxmbdhcemwld
krauabg