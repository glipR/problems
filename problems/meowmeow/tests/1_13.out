human noises
hiss
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
burble
burble
human noises
human noises
human noises
human noises
human noises
burble
human noises
hiss
human noises
trill
human noises
human noises
burble
human noises
human noises
human noises
human noises
hiss
human noises
human noises
human noises
human noises
burble
human noises
human noises
human noises
human noises
human noises
human noises
trill
human noises
human noises
hiss
trill
human noises
trill
human noises
human noises
human noises
human noises
human noises
human noises
hiss
human noises
burble
human noises
human noises
human noises
human noises
human noises
